network/p2p/config.go

```diff
@@ -63,7 +63,7 @@ type Config struct {
 	// OperatorPubKeyHash is hash of operator public key, used for identity, optional
 	OperatorPubKeyHash string
 	// OperatorID contains numeric operator ID
-	OperatorID spectypes.OperatorID
+	OperatorID func() spectypes.OperatorID
```

Comment:
Please update comment. 

Also, I'd consider renaming to something like `OperatorIDFunc`/`OperatorID`/`GetOperatorID`/`OperatorIDGetter` to explicitly show that this is not a number